mainwindow.cpp

```diff
@@ -64,10 +64,8 @@ MainWindow::MainWindow(QSettings &settings)
     m_ui->mainToolBar->insertWidget(m_ui->actionOpen_Password, sessionMenuButton);
     */
 
-    // TODO: Could we make connections easier?
     // We connect John process' signals with our slots.
     // John was ended.
-    // TODO: It will good to show exit status and exit code to user.
```

Comment:
On Wed, May 13, 2015 at 06:50:03AM -0700, math07 wrote:
> >      // We connect John process' signals with our slots.
> >      // John was ended.
> > -    // TODO: It will good to show exit status and exit code to user.
> 
> I think it's too complex information for the user. Most users won't understand what exit code mean.

Exit with value != 0 may provide a message like "John the Ripper
exited abnormally". "exited abnormally" seems to be a usual phrase
seen in programs.

The message may be similar when johnny thinks that john crashed.

I think john does not use exit codes to pass a lot of information,
just good or not good (if at all).

Thanks!

-- 
Regards,
Aleksey Cherepanov